livestream/ttl_counter.go

```diff
@@ -24,7 +38,6 @@ func (swc *SlidingWindowCounter) Increment() {
 
 	now := time.Now()
 	swc.events = append(swc.events, now)
-	swc.removeOldEvents(now)
```

Comment:
This was going to be unnecessarily thrashy - let's just do it on a background goroutine.